packages/core/strapi/src/services/metrics/__tests__/admin-user-hash.test.ts

```diff
@@ -1,6 +1,6 @@
 import crypto from 'crypto';
+import createContext from 'strapi/test/helpers/create-context';
```

Comment:
Just curious why this change is here? It's intentional?

Reply:
Not idea how this get here 🤔 I will remove it